backend/src/environment.ts

```diff
@@ -33,6 +37,8 @@ export class Environment {
   deprecatedBinaries: string[]
   manufacturerDetails: ManufacturerDetails
   version: string
+  oobAvailable: boolean
+  oobActive: boolean
```

Comment:
We don't want these in the environment file. They are destined for the LAN.ts class.

Reply:
moved to lan class